### PR TITLE
NXP-27348: make user provisioning steps include email on webUI ft

### PR DESCRIPTION
**Pull request by FabioFSantos**

### CODE REVIEW

**Comment by FabioFSantos** [comment]
https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-fdsantos/34/ 🎉

File: packages/nuxeo-web-ui-ftest/test/features/step_definitions/search.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ Given(/^I have the following users$/, table => Promise.all(
     'entity-type': 'user',
     properties: {
       username: row[0],
+      email: `${row[0]}@test.com`,
```
Comment (Gabez0r) [inline packages/nuxeo-web-ui-ftest/test/features/step_definitions/search.js]: Perhaps we should read the email from the table as well?

Comment (FabioFSantos) [reply packages/nuxeo-web-ui-ftest/test/features/step_definitions/search.js]: https://github.com/nuxeo/nuxeo-web-ui/pull/640/commits/dfe04e7130d0cfe13dfa28966787aae17c5c1465 👍

**Comment by FabioFSantos** [comment]
https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-fdsantos/39/ 🎉

**Pull request merged**